.test-infra/jenkins/job_beam_PostCommit_Java_ValidatesRunner_Dataflow.groovy

```diff
@@ -41,5 +41,18 @@ mavenJob('beam_PostCommit_Java_ValidatesRunner_Dataflow') {
     'Run Dataflow ValidatesRunner')
 
   // Maven goals for this job.
-  goals('-B -e clean verify -am -pl runners/google-cloud-dataflow-java -DforkCount=0 -DvalidatesRunnerPipelineOptions=\'[ "--runner=TestDataflowRunner", "--project=apache-beam-testing", "--tempRoot=gs://temp-storage-for-validates-runner-tests/" ]\'')
+  goals('''clean verify
```

Comment:
As mentioned about making this an array instead

Reply:
Done